src/main/features/response/views/evidence.njk

```diff
@@ -59,6 +89,7 @@
           {{ evidenceRow(loop.index0, form) }}
         {% endfor %}
         </div>
+        <br />
```

Comment:
do not use br, use css instead, prototype does this for hacking things together quickly